SMARTAGRIFOOD/wineTraceability/grapeTransportContainer/CONTRIBUTORS.YAML

```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of all CONTRIBUTORS across different objects (data models) alphabetically ordered. All fields are non mandatory
```

Comment:
file is correct the location not because it goes at the root of the subject. anyhow don't worry.